app/helpers/current_question_helper.rb

```diff
@@ -1,10 +1,31 @@
 module CurrentQuestionHelper
   def calculate_current_question_path(presenter)
+    if presenter.use_session
+      calculate_session_question_path(presenter)
+    else
+      calculate_current_smart_answers_question_path(presenter)
+    end
+  end
+
+  def calculate_current_smart_answers_question_path(presenter)
     attrs = params.permit(:id, :started).to_h.symbolize_keys
     attrs[:responses] = presenter.accepted_responses if presenter.accepted_responses.any?
     smart_answer_path(attrs)
   end
 
+  def calculate_session_question_path(presenter)
+    node_name = presenter.current_state.current_node.to_s
+    node_name.delete_suffix!("?")
+    update_session_flow_path(id: params[:id], node_name: node_name)
+  end
+
+  def restart_flow_path(presenter)
+    flow_name = params[:id]
+    flow_name.gsub!(/_/, "-") if presenter.use_session
+
+    smart_answer_path(flow_name)
+  end
```

Comment:
Do we have tests for these methods?

Reply:
I'll add them.

Reply:
I've added a helper test to test the methods modified here. However, in doing so I found a support module that was erroneously placed in test/helpers instead of test/support. When I looking in test/support I found there was already a module there that contained similar methods, so I removed the test/helpers module and moved its methods to the existing test/support module.